x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -67,44 +67,60 @@ export const callEnterpriseSearchConfigAPI = async ({
       publicUrl: stripTrailingSlash(data?.settings?.external_url),
       readOnlyMode: !!data?.settings?.read_only_mode,
       ilmEnabled: !!data?.settings?.ilm_enabled,
+      isFederatedAuth: !!data?.settings?.is_federated_auth, // i.e., not standard auth
       configuredLimits: {
-        maxDocumentByteSize: data?.settings?.configured_limits?.max_document_byte_size,
-        maxEnginesPerMetaEngine: data?.settings?.configured_limits?.max_engines_per_meta_engine,
+        appSearch: {
+          engine: {
+            maxDocumentByteSize:
+              data?.settings?.configured_limits?.app_search?.engine?.document_size_in_bytes,
+            maxEnginesPerMetaEngine:
+              data?.settings?.configured_limits?.app_search?.engine?.source_engines_per_meta_engine,
+          },
+        },
```

Comment:
Note to self:

There's a couple other configured limits for App Search that Bmac added to this endpoint:

<img width="362" alt="Screen Shot 2020-09-08 at 11 49 39 AM" src="https://user-images.githubusercontent.com/549407/92516027-6bf28d80-f1c9-11ea-8cea-2b3ef269fe17.png">

I left them out for now because the App Search UI doesn't indicate using/displaying those values (at least in AppLogic), but they're there if we ever need to ✨